packages/mobile/src/invite/RegulatoryTerms.tsx

```diff
@@ -0,0 +1,111 @@
+import Button, { BtnTypes } from '@celo/react-components/components/Button'
+import fontStyles from '@celo/react-components/styles/fonts'
+import * as React from 'react'
+import { Trans, WithTranslation } from 'react-i18next'
+import { ScrollView, StyleSheet, Text, View } from 'react-native'
+import SafeAreaView from 'react-native-safe-area-view'
+import { connect } from 'react-redux'
+import { acceptTerms } from 'src/account/actions'
+import { componentWithAnalytics } from 'src/analytics/wrapper'
+import DevSkipButton from 'src/components/DevSkipButton'
+import { CELO_TERMS_LINK } from 'src/config'
+import { Namespaces, withTranslation } from 'src/i18n'
+import { nuxNavigationOptions } from 'src/navigator/Headers'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { RootState } from 'src/redux/reducers'
+import { navigateToURI } from 'src/utils/linking'
+
+function mapStateToProps(state: RootState) {
+  return {}
+}
+
+interface DispatchProps {
+  acceptTerms: typeof acceptTerms
+}
+
+const mapDispatchToProps: DispatchProps = {
+  acceptTerms,
+}
+
+type Props = WithTranslation & DispatchProps
+
+export class RegulatoryTerms extends React.Component<Props> {
+  static navigationOptions = nuxNavigationOptions
+
+  onPressAccept = () => {
+    this.props.acceptTerms()
+    navigate(Screens.EnterInviteCode)
+  }
+
+  onPressGoToTerms = () => {
+    navigateToURI(CELO_TERMS_LINK)
+  }
+
+  render() {
+    const { t } = this.props
+
+    return (
+      <SafeAreaView style={styles.container}>
+        <DevSkipButton nextScreen={Screens.PincodeEducation} />
+        <ScrollView contentContainerStyle={styles.scrollContainer}>
+          <View style={styles.terms}>
+            <Text style={fontStyles.h1} testID="Terms">
+              {t('terms.title')}
+            </Text>
+            <Text style={styles.header}>{t('terms.heading1')}</Text>
+            <Text style={styles.disclaimer}>
+              <Trans ns={Namespaces.nuxNamePin1} i18nKey={'terms.privacy'}>
```

Comment:
use `Trans` rather than two separate `i18n.t `calls as this allows for languages with different word order typology